webpack.config.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
   devServer: {
     contentBase: path.resolve(__dirname, 'examples'),
     publicPath: '/',
+    // historyApiFallback: {
+    //   rewrites: [{ from: /\/hooks\/*/, to: '/hooks/index.html' }],
+    // },
```

Comment:
🤔

Reply:
When reloading pages, we get `404` from webpack-dev-server 'cos those URLs (`/hooks/use-query-param`) don't exist and it was a bit of an inconvenience. This hack let's us use the reload workflow but prevents `contentBase` serving the root examples we generate. I left the comment here so I can uncomment it just for hooks development.

Reply:
It can be just `historyApiFallback: true`, to fallback to `/index.html` and let "router" handle the rest in a "SPA mode"

Reply:
`historyApiFallback: true` doesn't work when we generate and serve static files as a directory using `contentBase` option in `devServer`. It always gives 404. Haven't figured why it does that though.